homework/Vasili_Zasinets/MagicBallTask/src/main/java/com/tutrit/java/quickstart/homework/ball/Ball.java

```diff
@@ -0,0 +1,36 @@
+package com.tutrit.java.quickstart.homework.ball;
+
+public enum Ball {
```

Comment:
no need to create enum for that. Prediction collection or map was a good decision.
let's revise this on next techTalk using SOLID?